launch/launch/actions/execute_process.py

```diff
@@ -378,10 +379,19 @@ def get_sub_entities(self):
             return self.__on_exit
         return []
 
+    def __deferred_shutdown(self, event, context):
+        return self._shutdown_process(context, send_sigint=True)
```

Comment:
@Blast545 I'd think `send_sigint` has to have the same value it had when `_shutdown_process()` was first called.

Reply:
I agree, I'll add a commit to address this.